src/main/java/com/atlassian/db/replica/api/aurora/AuroraReplicasDiscoverer.java

```diff
@@ -0,0 +1,55 @@
+package com.atlassian.db.replica.api.aurora;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.atlassian.db.replica.api.aurora.AuroraEndpoints.replicaUrl;
+
+/**
+ * It allows to pull Aurora Replicas cluster information
```

Comment:
Why is it needed?

Reply:
Once you have instance endpoints you can build connection pool per replica.